src/WebJobs.Script/Content/Script/http/response.js

```diff
@@ -4,6 +4,7 @@
 module.exports = (context) => {
     var res = {
         headers: {},
+        body: undefined,
```

Comment:
It's subtle, the property has value undefined but is not 'undefined'.  Therefore when parsing json in C# we can recognize that there is a body property and flow through the correct response handling path.